### ISSUE TITLE
Add labels to font size buttons for screen readers

### ISSUE THREAD

**Issue by aslagle** [NYPL-Simplified/webpub-viewer#47]

**Comment by ricardoom** [comment]
Looks like these are already on there:
https://github.com/NYPL-Simplified/webpub-viewer/blob/master/src/BookSettings.ts#L27
https://github.com/NYPL-Simplified/webpub-viewer/blob/master/src/BookSettings.ts#L33

the `aria-labelledby` should be reading back the control...

cc: @wlla 

Unless I've implemented this incorrectly. :thinking:

**Comment by ricardoom** [comment]
this should be already fixed...
